### PR TITLE
Trigger a view to be loaded before first call of `traitCollectionDidChange`

### PR DESCRIPTION
**Pull request by yas375**
Based on [this conversation](https://github.com/ashfurrow/Nimble-Snapshots/pull/117#issuecomment-311215540). /cc @marcelofabri

### CODE REVIEW

File: Bootstrap/BootstrapTests/DynamicTypeTests.swift [hunk 1]
```diff
@@ -81,6 +81,13 @@ class DynamicTypeTests: QuickSpec {
 
                 expect(view) == dynamicTypeSnapshot()
             }
+
+            it("works with view controller from storyboard") {
```
Comment (marcelofabri) [inline Bootstrap/BootstrapTests/DynamicTypeTests.swift]: can you just rename the test to say it's related to `traitCollectionDidChange`?

**Comment by yas375** [comment]
@marcelofabri thanks for the review 👍 Just force pushed with updated changelog and renamed test.

**Comment by orta** [comment]
Cool - looks good @marcelofabri do you have trunk access to push this?

**Comment by marcelofabri** [comment]
No 😢

**Comment by ashfurrow** [comment]
@marcelofabri Would you like access? Email me at ash@ashfurrow.com with whatever email you'd like.

**Comment by marcelofabri** [comment]
This is now available as part of [6.2.0](https://github.com/ashfurrow/Nimble-Snapshots/releases/tag/6.2.0) 🎉 

Thanks again @yas375 for your contribution ✅

**Pull request merged**